code/game/machinery/doors/poddoor.dm

```diff
@@ -22,7 +23,7 @@
 /obj/machinery/door/poddoor/attackby(obj/item/I, mob/user)
 	add_fingerprint(user)
 
-	if(!istype(I, /obj/item/weapon/crowbar))
+	if(!istype(I, /obj/item/weapon/crowbar) && !istype(I, /obj/item/weapon/twohanded/fireaxe))

```

Comment:
This is a bit confusing. I believe it would be clearer to do something like this:
```
if(istype(I, /obj/item/weapon/twohanded/fireaxe))
    var/obj/item/weapon/twohanded/fireaxe/F = I
    if(!F.wielded)
        return
else if(!istype(I, /obj/item/weapon/crowbar))
    return
```
This way you can avoid checking for fireaxe twice.